src/sql/workbench/parts/dataExplorer/electron-browser/nodeActions.contribution.ts

```diff
@@ -18,24 +19,34 @@ MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
 	when: NodeContextKey.IsConnected
 });
 
+// The weird regex is because we want this to generically apply to Database and Server nodes but right now
```

Comment:
We control the value of viewitem for nodes coming from the "data sourcing" part of the code. So we can make it so the viewitem value for mssql nodes in extension trees avoid potentially confusing strings such as "serverlevellogin"